src/components/queryBuilder/LogLevelField.tsx

```diff
@@ -7,19 +7,14 @@ import { FullField } from 'types';
 interface LogLevelEditorProps {
   fieldsList: FullField[];
   onLogLevelFieldChange: (logLevelField: string) => void;
+  logLevelField: string | null;
```

Comment:
```suggestion
  logLevelField: string;
```
I believe strings are already nullable. Does this work with just string?

Reply:
No, null is not assignable to type string, that's why it's explicit here.